File: src/Java/tasks/SampleCalculator.java
package Java.tasks;

import java.util.Scanner;

public class SampleCalculator {
    public static void main(String[] args) throws Exception {


        Scanner input = new Scanner(System.in);

        System.out.println("Enter two number : ");
        double num1 = input.nextDouble();
        double num2 = input.nextDouble();

        System.out.println("Enter your Operator");
        char operator = input.next().charAt(0);

        System.out.println("num1 = " + num1);
        System.out.println("num2 = " + num2);
        System.out.println("operator = "+ operator);

        double result = 0;
       switch (operator) {
           case '+' :
              result =  num1 + num2;
              break;
           case  '-' :
               result = num1 - num2;
               break;
           case  '/':
               result = num1 / num2;
               break;
           case '*':
               result = num1 * num2;
               break;
           default:
               throw new Exception("Operator has to be +, -, *, or /");

       }
        System.out.println("Calculation is : " + result);
    }



}

File: python/tasks/sample_calculator.py
num1 = float(input("Enter the first number: "))
num2 = float(input("Enter the second number: "))
operator = input("Enter your operator (+, -, *, /): ")

print(f"num1 = {num1}")
print(f"num2 = {num2}")
print(f"operator = {operator}")

result = 0

if operator == '+':
    result = num1 + num2
elif operator == '-':
    result = num1 - num2
elif operator == '/':
    if num2 == 0:
        raise ZeroDivisionError("Division by zero is not allowed")
    result = num1 / num2
elif operator == '*':
    result = num1 * num2
else:
    raise ValueError("Operator has to be +, -, *, or /")

print(f"Calculation is: {result}")